src/pages/DetailBlog/DetailBlog.tsx

```diff
@@ -67,10 +63,17 @@ const DetailBlog = () => {
     } else {
       dispatch(showPopup('login'))
     }
-  }
+  }, [blogDetail])
```

Comment:
```suggestion
  }, [blogDetail, setIsLikeBlog])
```
if we not watch setIsLikeBlog, prev will be the same